src/app-simplefin/app-simplefin.js

```diff
@@ -0,0 +1,236 @@
+import express from 'express';
+import { inspect } from 'util';
+import https from 'https';
+import { SecretName, secretsService } from '../services/secrets-service.js';
+
+const app = express();
+export { app as handlers };
+app.use(express.json());
+
+app.post('/status', async (req, res) => {
+  let configured = false;
+
+  let token = secretsService.get(SecretName.simplefin_token);
+  if (token !== null && token !== undefined && token !== 'Forbidden') {
```

Comment:
These checks can be simplified to just `token != null` which check for both null and undefined.